api/envoy/extensions/filters/http/admission_control/v3alpha/admission_control.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.admission_control.v3alpha;
+
+import "envoy/config/core/v3/base.proto";
+import "envoy/type/v3/range.proto";
+
+import "google/api/annotations.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+import "google/rpc/status.proto";
+
+import "udpa/annotations/migrate.proto";
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.admission_control.v3alpha";
+option java_outer_classname = "AdmissionControlProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).work_in_progress = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Admission Control]
+// [#extension: envoy.filters.http.admission_control]
+
+message AdmissionControl {
+  // Default method of specifying what constitutes a successful request. All status codes that
+  // indicate a successful request must be explicitly specified if not relying on the default
+  // values.
+  message DefaultEvaluationCriteria {
+    // If HTTP statuses are unspecified, defaults to all non-5xx. Valid codes are in the range [100,
+    // 600).
+    repeated type.v3.Int32Range http_success_status = 1;
+
+    // GRPC status codes to consider as request successes. If unspecified, defaults to: Ok,
```

Comment:
Looking at the code unspecified really means empty. Wondering if should make the API a bit more explicit around what unspecified means by wrapping these in messages:

```
message EvaluationCriteria {
  message GrpcCriteria {
    repeated uint32 grpc_success_status = 1;
 }

  GrpcCriteria grpc_criteria = 1;
}
```

with the default being used if `grpc_criteria` is not set. Same thing for the list of HTTP ranges. 

The thinking here is that nobody will be surprised that setting an empty repeated (e.g. if configuring through yaml/json) is treated as it being unspecified. 

WDYT?

Reply:
That's a good point and I think it's a better approach. I'll make the change in the next patch set and we can see how it's looking.